kolibri/core/assets/src/views/k-radio-button.vue

```diff
@@ -62,14 +64,21 @@
         required: true,
       },
       /**
-       * v-model value
+       * Description for Label
+       */
+      description: {
```

Comment:
We need to also update the radio button guidelines to include this new description.